readme.md

```diff
@@ -104,6 +104,7 @@ GitHub Enterprise is also supported by [authorizing your own domain in the optio
 
 ### New Features
 
+- [Collapse/expand all outdated pull request comments by pressing <kbd>Alt</kbd> while clicking](https://user-images.githubusercontent.com/25818354/33240033-3e271588-d2af-11e7-93af-13b6e325f65d.gif) 
```

Comment:
You reset my change from https://github.com/sindresorhus/refined-github/pull/821/commits/d849b05c6ffd6e76e865ae41e08ef6cf81f6485c

Reply:
@sindresorhus, I saw that commit and it made me realize that I need to update the readme:
* The key combination is now only Alt + click for both Mac and Windows/Linux, which was suggested in the discussion: https://github.com/sindresorhus/refined-github/pull/821#discussion_r152041694
* I linked to the newest version of the GIF (which I also linked at the top of this pull request)
* I kept your other wording changes